src/main/java/org/mitre/synthea/export/CCDAExporter.java

```diff
@@ -117,6 +119,8 @@ public static String export(Person person, long time) {
       }
     }
 
+    person.attributes.put("wellness_provider", person.preferredProviders.getWellnessProvider());
```

Comment:
"wellness_provider" Magic String, would prefer a static final string from somewhere.

Reply:
Not critical though.

Reply:
It is a magic string, however it's not used in any other java code.

In context, a bunch of properties get set in the method and there are a ton of magic strings preceding it. I can extract them all out to constants, but IMHO, it doesn't add much value.